src/status_im/ui/screens/chat/views.cljs

```diff
@@ -341,44 +341,28 @@
 
 (def ^:const initial-limit 5)
 (def ^:const second-initial-limit-step 6)
-(def second-limit (+ initial-limit second-initial-limit-step))
 (def ^:const third-initial-limit-step 11)
 (def ^:const default-limit-step 20)
 (def ^:const initial-threshold 0.5)
 (def ^:const default-threshold 2)
 
-(defonce messages-limit (reagent/atom initial-limit))
-
-(defn increment-limit [lim]
-  (+ lim
-     (case lim
-       initial-limit second-initial-limit-step
-       second-limit third-initial-limit-step
-       default-limit-step)))
-
 (defview messages-view
   [{:keys [group-chat chat-id pending-invite-inviter-name contact] :as chat}
    modal?]
   (letsubs [messages           [:chats/current-chat-messages-stream]
             photo-path         [:chats/photo-path chat-id]
             current-public-key [:account/public-key]]
-    {:component-will-mount
-     (fn []
-       (reset! messages-limit initial-limit))
-     :component-did-mount
+    {:component-did-mount
      (fn [args]
        (when-not (:messages-initialized? (second (.-argv (.-props args))))
          (re-frame/dispatch [:chat.ui/load-more-messages]))
        (re-frame/dispatch [:chat.ui/set-chat-ui-props
                            {:messages-focused? true
                             :input-focused?    false}]))}
     (let [no-messages (empty? messages)
-          m-limit     @messages-limit
-          threshold   (if (= m-limit initial-limit)
-                        initial-threshold
-                        default-threshold)
+          threshold   default-threshold
```

Comment:
can be removed

Reply:
why is that? this threshold is 2 which is very different from theshold of 0.5 that is default in RN.

Reply:
you can put the threshold directly in the parameters of the flat list since it is not computed anymore

Reply:
oh, sorry, the `default-threshold` is 0.5, will remove.